packages/engine-server/src/drivers/file/index.ts

```diff
@@ -1 +1,2 @@
 export * from "./schemaParser";
+export * from "./parseAllNoteFilesForSqlite";
```

Comment:
Odd name for a file lol. Most files I've seen are nouns. On purpose?

Reply:
Intentional - per our new-ish convention (follows airbnb style guide), 1 export per file only, and name the file exactly the same as the export. So in the case of a class, which most files are, it'll be a noun. But for export functions, it should match the name of the function. Makes it easier to locate where things are. 

https://docs.dendron.so/notes/773e0b5a-510f-4c21-acf4-2d1ab3ed741e/#one-export-per-file